src/transformers/trainer_utils.py

```diff
@@ -425,6 +425,6 @@ def stop_and_update_metrics(self, metrics=None):
 
 class ShardedDDPOption(ExplicitEnum):
     SIMPLE = "simple"
-    ZERO_DP_2 = "zero2"
-    ZERO_DP_3 = "zero3"
-    OFFLOAD = "offload"
+    ZERO_DP_2 = "zero_dp_2"
```

Comment:
Thank for catching those discrepancies.

Ideally let's try not to cross unrelated areas in the same PR, but I suppose it's OK for now.